packages/calendar/src/components/input-types/date-input/DateInput.tsx

```diff
@@ -77,21 +78,26 @@ export const DateInput: React.FC<DateInputProps> = ({
     showCalendar,
   } = useDateInput(onChange, onClose, openOnMount);
 
+  const { tippyRef, onChangePanel } = usePopoverCalendar();
+
   return (
     <Box width={width}>
       <Popover
         arrow={false}
         lazy
         visible={showingCalendar}
         onClickOutside={hideCalendar}
+        placement={"bottom"}
```

Comment:
Why force this?

Reply:
It is not forced, it is just the default.
If it can not be fitted to the bottom, it will align in some other way.
But I thought that the default should be defined, and consistent.

Reply:
No, it can jump between over and under :) That's why I added it.